### PR TITLE
Euroscipy2019

### PR DESCRIPTION
**Pull request by Didou09**
The talk has been done, this is the final version of the presentation

### CODE REVIEW

**Comment by Didou09** [comment]
You're right, I added compiled files by mistake yesterday,
Thanks for removing them

File: benchmarks/calc_signal_benchmark.py [hunk 1]
```diff
@@ -42,9 +42,9 @@
 # Defining defaults
 ###################
 
-_LRES = [-1,-3,0]
-_LLOS = [1,3,0]
-_LT = [1,3,0]
+_LRES = [-3,-3,0]
+_LLOS = [5,5,0]
+_LT = [3,3,0]
```
Comment (lasofivec) [inline benchmarks/calc_signal_benchmark.py]: this is probably shouldn't be part of the PR. We can leave it if you want @Didou09, but for cleaningness maybe you want to remove it ?

**Comment by Didou09** [comment]
Any more changes ? I'm gonna accept the pull request

**Comment by lasofivec** [comment]
You can ofc :) but I did do a comment in the code (see in this same thread above).

File: benchmarks/calc_signal_benchmark.py (hunk 1 above)
Comment (Didou09) [reply benchmarks/calc_signal_benchmark.py]: Hum, not sure, it's good to have easy access to typical becnhmark input

Comment (Didou09) [reply benchmarks/calc_signal_benchmark.py]: ok
